ksql-engine/src/main/java/io/confluent/ksql/function/udaf/count/CountKudaf.java

```diff
@@ -32,7 +32,8 @@
 
   CountKudaf(String functionName, int argIndexInValue) {
     super(functionName, argIndexInValue, () -> 0L, Schema.OPTIONAL_INT64_SCHEMA,
-          Collections.singletonList(Schema.OPTIONAL_FLOAT64_SCHEMA)
+          Collections.singletonList(Schema.OPTIONAL_FLOAT64_SCHEMA),
+        "counts records by key"
```

Comment:
nit: indentation. Also, if this is displayed to the user we probably should include proper punctuation (capitalization, periods, etc.).